libs/application/templates/parental-leave/src/lib/messages.ts

```diff
@@ -791,6 +791,16 @@ export const mm: MessageDir = {
     },
   }),
   confirmation: defineMessages({
+    epxandAll: {
+      id: 'pl.application:confirmation.buttons.expandAll',
+      defaultMessage: 'Auka allt',
```

Comment:
I am actually unsure what the right translation for `Expand` should be, but it is not `auka`. 

I propose: "Stækka allt" or "Sýna allt" and for collapse I would go for "Fela allt" or "Minnka allt"

Reply:
Perfect thanks! I will update it now.

Reply:
Should be fixed now.